aws/resource_aws_elasticache_replication_group.go

```diff
@@ -165,6 +187,18 @@ func resourceAwsElasticacheReplicationGroupCreate(d *schema.ResourceData, meta i
 		params.SnapshotName = aws.String(v.(string))
 	}
 
+	if _, ok := d.GetOk("transit_encryption"); ok {
+		params.TransitEncryptionEnabled = aws.Bool(d.Get("transit_encryption").(bool))
+	}
+
+	if _, ok := d.GetOk("at_rest_encryption"); ok {
```

Comment:
Likewise - would you mind changing it to `at_rest_encryption_enabled` for the same reasons as the above field?

Reply:
👍